File: app/src/main/java/com/neome/java/api/home/main/sig/SigGroupMessageCandidateList.java
// neome.ai API - do not change
//

package com.neome.java.api.home.main.sig;

import com.neome.java.api.home.drawer.sig.SigGroupAvatar;
import com.neome.java.api.nucleus.base.Types.AnyPrefixKey;
import com.neome.java.api.nucleus.base.sig.Sig;

import java.util.Map;

@SuppressWarnings("unused")
public class SigGroupMessageCandidateList extends Sig
{
  public Map<AnyPrefixKey, SigGroupAvatar[]> candidateMap;
}

File: app/src/main/java/com/neome/api/home/main/sig/SigGroupMessageCandidateList.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.main.sig

import com.neome.api.nucleus.base.Types.AnyPrefixKey
import com.neome.api.nucleus.base.sig.Sig
import com.neome.api.home.drawer.sig.SigGroupAvatar

interface SigGroupMessageCandidateList : Sig
{
  val candidateMap: Map<AnyPrefixKey, Array<SigGroupAvatar>>
}